src/test/java/uk/gov/companieshouse/overseasentitiesapi/service/PrivateDataRetrievalServiceTest.java

```diff
@@ -584,6 +615,18 @@ void testGetTrustDetailsApiErrorResponseExceptionThrownNotFoundReturnsEmptyList(
             assertEquals(0, result.getData().size());
         }
 
+        @Test
+        void testGetTrustLinksApiErrorResponseExceptionThrownNotFoundReturnsEmptyList() throws ApiErrorResponseException, URIValidationException, ServiceException {
+            var exception = new ApiErrorResponseException(
+                    new HttpResponseException.Builder(404, "notFound", new HttpHeaders()));
```

Comment:
Here you can use the constant `FOUR_HUNDRED_AND_FOUR_EXCEPTION` 

Looking at it there is actually two one incorrectly named `FourHundredAndFourException` if you could rename that one `FOUR_HUNDRED_AND_FOUR_EXCEPTION` then delete the original that would be great